### ISSUE TITLE
Scheduler framework cannot disable default Predicate function

### ISSUE THREAD

**Issue by angao** [kubernetes/kubernetes#85986]
**What happened**:

scheduler framework cannot disable default in-tree `Predicate` function:
I create the config file as follows:
```
apiVersion: kubescheduler.config.k8s.io/v1alpha1
kind: KubeSchedulerConfiguration
schedulerName: sample
plugins:
  filter:
    disabled:
    - name: "NodeResourcesFit"
```
I found the `NodeResourcesFit` FilterPlugin is also effective and not disabled by config.

**What you expected to happen**:

 I want to know how to disable the in-tree framework plugins, include filter, scoring

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): v1.17.0-rc.02
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by draveness** [comment]
/assign

I think we didn't consider the disable of the scheduling framework plugins, I could work on this and find a fix.

**Comment by angao** [comment]
I found this [Plugin Configuration](https://kubernetes.io/docs/concepts/configuration/scheduling-framework/#plugin-api). So far, it seems that is not implemented.

**Comment by draveness** [comment]
> I found this [Plugin Configuration](https://kubernetes.io/docs/concepts/configuration/scheduling-framework/#plugin-api). So far, it seems that is not implemented.

Yes, this is the missing part......

**Comment by ahg-g** [comment]
This is not really a bug, this is by design. 

We don't have "default plugins" yet, [see comment here](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/scheduler.go#L260). What we have is a priorities/predicates converted to plugins.

As part of [phase 2 of the migration](https://github.com/kubernetes/kubernetes/issues/85822), we will "Define algorithm providers in terms of plugins instead of predicates/priorities."

In summary, we still rely on the old "Policy" interface to schedule behavior until phase 2 is done.

**Comment by ahg-g** [comment]
That being said, the logic to disable plugins is certainly missing.

**Comment by angao** [comment]
thx! @ahg-g @draveness

**Comment by ahg-g** [comment]
This is now possible.

/close
